doc/conf.py

```diff
@@ -139,7 +139,8 @@
         'binderhub_url': 'https://mybinder.org',  # noqa: E501 Any URL of a binderhub deployment. Must be full URL (e.g. https://mybinder.org).
         'filepath_prefix': filepath_prefix,  # noqa: E501 A prefix to prepend to any filepaths in Binder links.
         'dependencies': [
-            './environment.yml',
+            '../requirements.txt',
+            './requirements.txt',
```

Comment:
note: the binder situation is a bit complicated. 

The changes here will make sure that the 2 requirements.txt files are getting copied to a `html/binder` directory upon build ... BUT they won't be used to build a docker image if a user clicks on our "binder links", because our docs are nested:

```
.
├── dev
├── environment.yml
├── index.html
├── mne_bids
├── mne_bids.egg-info
├── stable
├── v0.1
├── v0.2
├── v0.3
└── v0.4

```

and binder cannot reach e.g., the `v0.4/binder` directory. Instead, it would expect the `binder` directory at the root. 

Instead it will always use whatever file it finds instead. In this case the good old `environment.yml`.

I think that's fine, but it is unexpected for DEVs so I am making you aware now. x-ref: https://github.com/sphinx-gallery/sphinx-gallery/issues/628

cc @jasmainak @agramfort @hoechenberger

Reply:
practically this means that for a new release, maintainers should also have a look at the `environment.yml` file on the gh-pages branch ... it may need some version bumps for a given release.

I'll update our release docs in the WIKI.

Reply:
Thanks for updating the wiki!